src/lib/utils/__tests__/asyncStorage.js

```diff
@@ -0,0 +1,96 @@
+import AsyncStorage from '../asyncStorage'
+
+jest.mock('../asyncStorage', () => {
```

Comment:
this isn't actually i wanted. i wanted to mock the window.localStorage. So AsyncStorage library and asyncStorage .js module should actually working. This test have no value

Reply:
1. remove this mock ( lines 3 - 35)
2. install & connect this package https://www.npmjs.com/package/jest-localstorage-mock